src/c/profile/discovery/discovery.c

```diff
@@ -19,62 +19,65 @@
 
 typedef struct CallbackData
 {
-    uxrAgentAddress* chosen;
     uxrOnAgentFound on_agent;
     void* args;
 
 } CallbackData;
 
 static void write_get_info_message(ucdrBuffer* ub);
-static bool listen_info_message(uxrUDPTransportDatagram* transport, int period, CallbackData* callback);
+static void listen_info_message(uxrUDPTransportDatagram* transport, int period, CallbackData* callback);
 static bool read_info_headers(ucdrBuffer* ub);
```

Comment:
Why is it the only function that returns a boolean?

Reply:
Ok, I added a "boolean meaning" to the rest of the functions. In the future, we can interpret this value in order to improve some behaviour.